src/Ledger/Conway/Specification/Epoch.lagda

```diff
@@ -85,13 +85,20 @@ instance
   HasPParams-EpochState .PParamsOf = PParamsOf ∘ EnactStateOf
 \end{code}
 \begin{NoConway}
+
+`PoolDistr`{.AgdaBound} in the Shelley spec results from dividing the coins in
+`PoolDelegatedStake`{.AgdaBound} by the total stake in
+`PoolDelegatedStake`{.AgdaBound}.
+
 \begin{code}
+PoolDelegatedStake = KeyHash ⇀ (Coin × KeyHash)
```

Comment:
ATM we don't have VRF's as part of the Agda spec, so I'd suggest to remove it from the codomain of this type.

Reply:
I'm confused. Don't you need `poolVRF` in order to define the `calculatePoolDistr` function, as in the Shelley spec?  Is the absence of `VRF` explained by the comment you added in your last commit?  I'll invite you to the call with Carlos tomorrow so we can discuss this and get this PR approved.